packages/@aws-cdk/aws-rds/lib/instance.ts

```diff
@@ -710,7 +710,7 @@ export interface DatabaseInstanceSourceProps extends DatabaseInstanceNewProps {
    *
    * @default - default master key
    */
-  readonly secretKmsKey?: kms.IKey;
+  readonly secretEncryptionKey?: kms.IKey;
```

Comment:
call this `masterPasswordEncryptionKey` or something similar instead ?

Reply:
Done.